src/support_sphere/lib/presentation/components/auth/signup_form.dart

```diff
@@ -184,15 +219,21 @@ class _SignupCodeInput extends StatelessWidget {
           autovalidateMode: AutovalidateMode.onUserInteraction,
           /// Checks input for Signup code to be length of 7 characters
           /// and uppercase value
-          validator: (value) => validateValue<SignupCubit>(
+          validator: (value) {
+            final validateResult = validateValue<SignupCubit>(
             [
               FormBuilderValidators.required(),
               FormBuilderValidators.equalLength(7),
               FormBuilderValidators.uppercase(),
             ],
             value,
             context,
-          ),
+          );
+          if (validateResult != null) {
+            context.read<SignupCubit>().setInvalid();  
+          }
+          return validateResult;
```

Comment:
This looks the same like the `validateValue` function

```
if (validateResult != null) {
  cubit.setInvalid();
  return validateResult;
}
cubit.setValid();
return null;
```

seems like the only difference is that the `cubit.setValid` is no longer in that function but rather a different spot in the code. Could you update this logic to be in `validateValue` function instead?

Reply:
Oh yes, was experimenting changing the isValid in the form, didn't realize this!

Have updated now.